src/themes/default/globals/variation.variables

```diff
@@ -3,10 +3,42 @@
 ***********************************************************/
 
 /* General */
-@variationAllSizes: mini, tiny, small, large, big, huge, massive;
```

Comment:
i would prefer having one line per each setting in this file, please add  `// prettier-ignore` fpr all list variables in the general block. It gets a quicker overview what switches are available and i also fear people could have some CI which replaces single lines